src/client/api/location/mod.rs

```diff
@@ -51,6 +51,8 @@ pub struct GeoCoordinates {
 pub enum GeoServiceVerified {
     Verified,
     Unverified,
+    #[serde(rename = "address_coordinates")]
+    AddressCoordinates,
 }
```

Comment:
```suggestion
#[serde(rename_all = "snake_case")]
pub enum GeoServiceVerified {
    Verified,
    Unverified,
    AddressCoordinates,
}
```

You should be able to modify the `rename_all` attribute to `snake_case` and that'll then remove the need for the individual `rename` on the variant :)

Worth a try just to make sure it still solves your error though!

https://serde.rs/container-attrs.html#rename_all

Reply:
Yes I assumed that'd be sufficient but wasn't sure if you intended the lowercase for another reason. Will test now.